auk-notebook-domains.ipynb

```diff
@@ -0,0 +1,277 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<img src=\"https://raw.githubusercontent.com/archivesunleashed/archivesunleashed.org/master/themes/hugo-material-docs/static/images/logo-square.png\" height=\"100px\" width=\"100px\">\n",
+    "\n",
+    "\n",
+    "# Welcome to the Domains Analysis Notebook\n",
+    "\n",
+    "Welcome to the Archives Unleashed domains analysis Jupyter Notebook. This demonstration takes the domain derivatives from the Cloud and uses Python to analyze and produce information about your collection.\n",
```

Comment:
Let's capitalize Domains Analysis in `Welcome to the Archives Unleashed domains analysis Jupyter Notebook.` i.e. so that it reads `Welcome to the Archives Unleashed Domains Analysis Jupyter Notebook.`